src/main/resources/migration/changelog-v5.6.0.xml

```diff
@@ -1334,4 +1334,20 @@
             columnNames="NAME"
             constraintName="TEAM_NAME_IDX" />
     </changeSet>
+
+    <changeSet id="v5.6.0-18" author="sahibamittal">
+        <sql splitStatements="false">
+            CREATE TYPE severity AS ENUM (
+                'UNASSIGNED'
+                , 'INFO'
+                , 'LOW'
+                , 'MEDIUM'
+                , 'HIGH'
+                , 'CRITICAL'
+            );
+            CREATE CAST (varchar AS severity) WITH INOUT AS IMPLICIT;
```

Comment:
Is this required? Usually the cast should "just work" if the text value is equal to the enum's name.

Reply:
When I build it locally with new changeset (with existing data in my db), I was getting some cast exception. This resolved it.
Let me re-create the scenario to check it again.

Reply:
We should be good without creating `CAST` (must be some bogus data at last time of testing).